azure-pipelines.yml

```diff
@@ -107,25 +107,16 @@ jobs:
       packDirectory: $(Build.ArtifactStagingDirectory)/deployables
 
   - powershell: >-
```

Comment:
At this point we have no multi-line commands any more and could improve readability if we changed this `>-` to `|` and removed all the blank lines.